src/alice.ts

```diff
@@ -81,6 +89,21 @@ export class Alice implements IAlice {
     };
   }
 
+  public listen(
+    port: number = 80,
+    webhookUrl: string = '/',
+    options: object = {},
+  ): WebhookServer {
+    const server = new WebhookServer({
+      port: port,
+      webhookUrl: webhookUrl,
+      options: options,
+      handleRequest: this.handleRequest,
```

Comment:
Don't you loose `this` context here? Maybe `handleRequest: request => this.handleRequest(request)` or something like that?